client/package.json

```diff
@@ -23,20 +23,24 @@
     "url": "https://github.com/nvuillam/vscode-groovy-lint/issues"
   },
   "homepage": "https://github.com/nvuillam/vscode-groovy-lint#readme",
-  "scripts": {
-    "update-vscode": "vscode-install",
-    "postinstall": "vscode-install"
-  },
   "engines": {
-    "vscode": "^1.42.0",
+    "vscode": "^1.75.0",
     "node": ">=14.0.0"
   },
   "author": "Nicolas Vuillamy",
   "license": "GPL-3.0-only",
   "dependencies": {
-    "vscode-languageclient": "^6.1.1"
+    "@types/vscode": "^1.85.0",
+    "glob": "^10.3.10",
+    "typescript": "^5.3.3",
```

Comment:
Are you sure you need the full typescript dependency here, as the generated extension VSIX is in JS ?

Reply:
Maybe more in devDependencies N

Reply:
Maybe "@types/vscode" is devDependencies too ? :)

Reply:
> Are you sure you need the full typescript dependency here, as the generated extension VSIX is in JS ?

Will check on this, thanks or flagging.